Add note for windows platform users to manually add file templates to IDE (#1032)
square/workflow-kotlin: samples/tutorial/Tutorial1.md

Start by creating a new workflow and screen by creating a new file with the [file templates](../../install-templates.sh), adding it to the `tutorial-base` module:

___NOTE___

> _The install script would not work for windows platform. For windows user, please add the scripts from [file templates folder](../../fileTemplates) manually to Android Studio or Intellij Idea._

Reviewer: ```suggestion
_Windows OS users: Please add the scripts from [file templates folder](../../fileTemplates) directly to Android Studio or Intellij Idea._
```

Author: hi @steve-the-edwards fixed this note wording as suggested

> _Go to Settings --> Editor --> File and Code Templates --> Select Files Tab. Now click on + icon and copy the workflow file template content to editor and save it_

Reviewer: ```suggestion
_Go to Settings --> Editor --> File and Code Templates --> Select Files Tab. Now click on the '+' icon and copy the the content of the Workflow file template to the editor and save it._
```
